src/backend/base/langflow/custom/custom_component/component.py

```diff
@@ -444,7 +444,9 @@ def _process_connection_or_parameter(self, key, value) -> None:
     def _process_connection_or_parameters(self, key, value) -> None:
         # if value is a list of components, we need to process each component
         # Note this update make sure it is not a list str | int | float | bool | type(None)
-        if isinstance(value, list) and not any(isinstance(val, str | int | float | bool | type(None)) for val in value):
+        if isinstance(value, list) and not any(
+            isinstance(val, str | int | float | bool | type(None) | Message | Data) for val in value
+        ):
```

Comment:
I don't really understand this - can you check if it's a type of `Component` instead of not anything else?

Reply:
I did that earlier but it was giving errors.
This Function need further clean up later.